Assets/Tests/PlayModeTests/StarRatingTests.cs

```diff
@@ -204,8 +204,8 @@ public async void TestStarRating_EventQueueThreshold_Limit()
             Assert.IsNotNull(Countly.Instance.StarRating);
             Assert.AreEqual(0, Countly.Instance.StarRating._eventCountlyService._eventRepo.Count);
 
-            await Countly.Instance.Views.RecordCloseViewAsync("close_view");
-            Assert.AreEqual(1, Countly.Instance.Events._eventRepo.Count);
+            Countly.Instance.Views.StartView("view");
```

Comment:
same thing maybe here

Reply:
this was failing because of the event repo count after the update obviously, that test is pretty unnecessary to me but didn't want to change how it was working, just updated it into something that records a view and event